src/backend/distributed/utils/maintenanced.c

```diff
@@ -704,7 +706,10 @@ CitusMaintenanceDaemonMain(Datum main_arg)
 			timeout = Min(timeout, Recover2PCInterval);
 		}
 
-		/* the config value -1 disables the distributed deadlock detection  */
+		/*
+		 * Execute only on the maintenance database, if it configured, otherwise run from every daemon.
+		 * The config value -1 disables the distributed deadlock detection
```

Comment:
This comment changed, but the code stayed the same. Are there code changes missing here? Or were these changes reverted, but the comment not reverted? Or was this simply meant to clarify the comment? If the later, then it doesn't really clarify anything to me (e.g. what is the maintenance database? Is that the same as the main database?)

Reply:
> Or were these changes reverted, but the comment not reverted?

Yes, that: initially I [implemented](https://github.com/citusdata/citus/pull/7286/commits/b8d27b238b02d480fab01dc6190d96026f6aba25) the "main db" logic too because it seemed like a good fit, but @marcoslot [asked](https://github.com/citusdata/citus/pull/7286#issuecomment-1788645263) to move it to separate PR. Fixed the comment